utils/src/main/java/com/revenuecat/purchases/utils/Responses.kt

```diff
@@ -112,9 +114,9 @@ object Responses {
                     "management_url": null
                   }
                 }
-            """
+            """).toString()
```

Comment:
could make a string extension function `String.removeJsonFormatting()` for this to be clearer for future readers?